batchglm/train/numpy/base_glm/estimator.py

```diff
@@ -472,15 +504,18 @@ def finalize(self):
         transfers relevant attributes.
         """
         # Read from numpy-IRLS estimator specific model:
-        self._hessian = -self.model.fim.compute()
-        fisher_inv = np.zeros_like(self._hessian)
-        invertible = np.where(np.linalg.cond(self._hessian, p=None) < 1 / sys.float_info.epsilon)[0]
-        fisher_inv[invertible] = np.linalg.inv(-self._hessian[invertible])
-        self._fisher_inv = fisher_inv
-        self._jacobian = np.sum(np.abs(self.model.jac.compute() / self.model.x.shape[0]), axis=1)
-        self._log_likelihood = self.model.ll_byfeature.compute()
-        self._loss = np.sum(self._log_likelihood)
-
-    @abc.abstractmethod
+        self.model_container._hessian = -self.model_container.fim.compute()
+        fisher_inv = np.zeros_like(self.model_container._hessian)
+        invertible = np.where(np.linalg.cond(self.model_container._hessian, p=None) < 1 / sys.float_info.epsilon)[0]
+        fisher_inv[invertible] = np.linalg.inv(-self.model_container._hessian[invertible])
+        self.model_container._fisher_inv = fisher_inv
+        self.model_container._jacobian = np.sum(
+            np.abs(self.model_container.jac.compute() / self.model_container.x.shape[0]), axis=1
+        )
+        self.model_container._log_likelihood = self.model_container.ll_byfeature
+        self.model_container._loss = np.sum(self.model_container._log_likelihood)
+
     def get_model_container(self, input_data):
-        pass
+        """Deprecated: This is equivalent to self.model_container now and can be removed"""
+        # return Model(input_data=input_data)
+        return self.model_container
```

Comment:
Do we need this?

Reply:
left it in for now because this returned the model itself in the old API and I*m not sure if this would be a breaking change for diffxpy @davidsebfischer ?

Reply:
Followup: #136